src/components/dev-hub/button.js

```diff
@@ -187,7 +198,12 @@ const getArrow = ({ pagination, primary, secondary }) => {
  */
 
 const Button = ({ children, href, play, to, hasArrow = true, ...props }) => {
-    const isButton = !!(props.primary || props.secondary || play);
+    const isButton = !!(
+        props.primary ||
+        props.secondary ||
+        props.renderAsButton ||
```

Comment:
I added this new option for the `Button` so we can render anything as a button without needing to apply `primary` or `secondary` and then removing styles manually.